pkg/auth/scope/scope.go

```diff
@@ -48,11 +48,13 @@ func VerifyScope(ctx context.Context, scopeMap map[string]*authpb.Scope, resourc
 		for s, f := range supportedScopes {
 			if strings.HasPrefix(k, s) {
 				if valid, err := f(ctx, scope, resource, logger); err == nil && valid {
+					logger.Debug().Interface("scope", scope).Msg("scope is valid")
 					return true, nil
 				}
 			}
 		}
 	}
+	logger.Warn().Interface("scopemap", scopeMap).Interface("resource", resource).Msg("no valid scope found")
```

Comment:
To keep consistent with the previous, i would use Trace level here as well (a Warn is too much, as it's completely fine that the scope is not valid). I would also log the scope like the previous instead of `scopeMap`